### ISSUE TITLE
Irozuku Sekai no Ashita kara is wrongly spelled/not showing any episodes

### ISSUE THREAD

**Issue by viktorrko** [littleweeb/Desktop#11]
As the title says

 _Irozuku Sekai no Ashita kara_ is spelled as _Iro**d**uku Sekai no Ashita kara_

Also, the episodes of this anime are not showing. My guess is that the spelling mistake is causing the bug, but I am not sure.

**Comment by EldinZenderink** [comment]
Your guess is correct, but it aint a spelling mistake (which you probably figured out): https://kitsu.io/anime/iroduku-sekai-no-ashita-kara

That said, this is still an issue where not all synonyms/every possible title is used while searching for episodes, which is something that i am currently working on. I will open it up however, it will be more visisble as something that i need to test properly.

**Comment by viktorrko** [comment]
Yes you are right, I closed the issue because of the reason stated above, and also because it was a generally badly written issue report.
